### PR TITLE
Remove used of normalized option in tf.multinomial since it is not supported by tfjs-node.

### PR DESCRIPTION
**Pull request by adarob**
fyi, @tambien

### CODE REVIEW

File: music/src/core/data.ts [hunk 1]
```diff
@@ -190,7 +190,8 @@ export class DrumsConverter extends DataConverter {
 
   constructor(args: DrumsConverterArgs) {
     super(args);
-    this.pitchClasses = args.pitchClasses === undefined  ?
+    this.pitchClasses =
+        (args.pitchClasses === null || args.pitchClasses === undefined) ?
```
Comment (notwaldorf) [inline music/src/core/data.ts]: Does just `!args.pitchClasses ? ` work here?

File: music/src/music_rnn/model.ts [hunk 2]
```diff
@@ -317,21 +317,20 @@ export class MusicRNN {
...
-                  .as1D();
+        let logits = lastOutput.matMul(this.lstmFcW).add(this.lstmFcB).as1D();
+
+        let sampledOutput: tf.Tensor1D;
+        if (temperature) {
+          logits = logits.div(tf.scalar(temperature));
+          sampledOutput = tf.multinomial(logits, 1).as1D();
         } else {
-          sampledOutput = logits.argMax(1).as1D();
+          sampledOutput = logits.argMax().as1D();
```
Comment (notwaldorf) [inline music/src/music_rnn/model.ts]: wait, we don't want softmax here anymore?

File: music/src/core/data.ts (hunk 1 above)
Comment (adarob) [reply music/src/core/data.ts]: that's too easy

File: music/src/music_rnn/model.ts (hunk 2 above)
Comment (adarob) [reply music/src/music_rnn/model.ts]: softmax doesn't change the relative magnitudes so we don't need it before an argmax
